### DISCUSSION TITLE
Hide Cider from the dock/taskbar

### DISCUSSION THREAD

**Discussion by binaryben** [ciderapp/Cider#1346] [Feature Request]
Loving the app so far! Keen to see how it improves from here.

Since the app doesn't have contextual controls (play/pause from the app icon) or contextual information (album art as the icon) at the moment, it would be great if we can hide the icon from the dock or taskbar and move it to the menu bar/tray as a menulet.

**Comment by coredev-uk** [comment]
Enable close button hide and that should solve your problem.

**Comment by binaryben** [comment]
Not really a problem, but anyway. I've had a look in settings and can't find anything sounding like "Close button hide" (only [online reference to it I can find it this](https://github.com/ciderapp/Cider/issues/523)). Point me to the right direction please?

**Reply by KillerParticles** [reply]
> Not really a problem, but anyway. I've had a look in settings and can't find anything sounding like "Close button hide" (only [online reference to it I can find it this](https://github.com/ciderapp/Cider/issues/523)). Point me to the right direction please?

![image](https://user-images.githubusercontent.com/93286201/183252783-5226853c-8a6d-49a0-add3-a1e322571b2a.png)

**Comment by coredev-uk** [comment]
I believe it comes under the visual settings category.

**Comment by binaryben** [comment]
Ah good, I'm not crazy. It's not there as an option for me.

<img width="1225" alt="Screen Shot 2022-08-07 at 2 57 30 pm" src="https://user-images.githubusercontent.com/12827556/183279152-8ca35a78-6f10-4369-bf5e-118b6bcd3c55.png">

**Reply by coredev-uk** [reply]
Ah, yeah its disabled for mac

**Reply by binaryben** [reply]
Bugger. Can we please enable it in mac or is there a reason for it being gone?

**Reply by coredev-uk** [reply]
It's been disabled because of how mac handles windows, with windows being hidden on close (Cmd+W) and exited on quit (Cmd+Q). With this, hidden apps appear to remain in the taskbar on macOS. There is a function in Electron to hide the app but I'm unsure if its still supported.

**Reply by binaryben** [reply]
Not sure if it helps or not, but [Motrix](https://github.com/agalwood/Motrix) is another Electron app that behaves the way I'd expect it to in this regard (configurable so that I can click away to another app, and it disappears - icon lives in menu bar to bring it back). The source code there might provide some insight? It's hardly a must have feature though so I understand if it isn't prioritised 😄

**Reply by coredev-uk** [reply]
I can take a look into creating a setting for this, I'm thinking I might be able to replicate common settings with the application showing in the dock, menu bar or both. I'll let you know once this is implemented.
